### ISSUE TITLE
Workflow runs are not properly queued

### ISSUE THREAD

**Issue by mfbrantner** [GamesResearchTUG/Maroon#499]
When multiple workflow runs are requested in a short time frame, some are cancelled because of GitHub's weird implementation of the concurrency group feature.

More info in this comment:
https://github.com/GameLabGraz/Maroon/pull/490#issuecomment-2250206980

I'll submit a PR with a fix soon.

**Comment by FlorianGlawogger** [comment]
Still seems to happen, see [https://github.com/GameLabGraz/Maroon/pull/500#issuecomment-2260230896](https://github.com/GameLabGraz/Maroon/pull/500#issuecomment-2260230896)

**Comment by mfbrantner** [comment]
After some more trial and error, I think I might have found a better solution than trying to queue the workflow runs.

While trying some strategies to tackle this issue, I discovered that this problem only seems to occur when using a **professional** Unity license.

When I tried using my **personal** license, I was able to do 30 simultaneous jobs without any issues!

I did not find any official documentation of this behaviour, but saw some other people reporting similar results.

I'd be happy to provide my Unity credentials to someone who has the permissions to edit the secrets for this repo, since I do not use my personal license anyway.
